host/xtest/xtest_10000.c

```diff
@@ -280,7 +280,7 @@ static const uint8_t pbkdf2_3_dkm[] = {
 #define pbkdf2_4_password pbkdf2_1_password
 #define pbkdf2_4_salt pbkdf2_1_salt
 #define pbkdf2_4_iteration_count 16777216
-static const uint8_t pbkdf2_4_dkm[] = {
+static const uint8_t pbkdf2_4_dkm[] __maybe_unused = {
```

Comment:
Actually leaving out `-dM` reveals (gcc5.x here, same on 6.x)
```c
# 1 "/mnt/sshd/devel/optee_projects/qemu/build/../optee_os/out/arm/export-ta_arm32/host_include/compiler.h" 1                                                                                
# 29 "/mnt/sshd/devel/optee_projects/qemu/build/../optee_os/out/arm/export-ta_arm32/host_include/compiler.h"                                                                                 
#define COMPILER_H                                                                                           
...
#define __maybe_unused __attribute__((unused))
```

So it comes from:
`./optee_os/out/arm/export-ta_arm32/host_include/compiler.h`